internal/server/kong/ws/payload.go

```diff
@@ -107,7 +107,7 @@ func (p *Payload) configForVersion(version string) (cacheEntry, error) {
 		if err != nil {
 			p.logger.Error("failed to process config table update",
 				zap.Error(err),
-				zap.String("kong-dp-version", version),
+				zap.String("kong_dp_version", version),
```

Comment:
We can leave the Zap labels alone, as we do stick the `kebab-case` there.

Reply:
Fixed